src/writer.js

```diff
@@ -67,7 +67,10 @@ define([
             jrm = "http://dev.commcarehq.org/jr/xforms";
         }
 
-        uuid = form.formUuid; //gets set at parse time/by UI
+        uuid = form.formUuid;
+        if(!uuid) {
+            uuid = "http://openrosa.org/formdesigner/" + util.generate_xmlns_uuid();
```

Comment:
Good point. Will do in separate PR. For this one I'm just bringing back old functionality https://github.com/dimagi/Vellum/commit/3bca6cb8d6c05ec25736ca3cad7d71311f9d8f69#diff-7dec243e7443d2aa55101241b38a8575L71